kore/src/Kore/ASTVerifier/AttributesVerifier.hs

```diff
@@ -80,15 +77,12 @@ verifySortHookAttribute
     => AttributesVerification declAtts axiomAtts
     -> Attributes
     -> error Hook
-verifySortHookAttribute =
-    \case
-        DoNotVerifyAttributes ->
-            \_ -> return emptyHook
-        VerifyAttributes _ _ -> \attrs -> do
-            hook <- parseAttributes attrs
-            case getHook hook of
-                Just _  -> return hook
-                Nothing -> koreFail "missing hook attribute"
+verifySortHookAttribute (VerifyAttributes _ _) =
+    \attrs -> do
+        hook <- parseAttributes attrs
+        case getHook hook of
+            Just _  -> return hook
+            Nothing -> koreFail "missing hook attribute"
```

Comment:
I don't think you need to use a lambda here, you can just add the `attrs` argument with the others.